packages/ui/src/common/components/forms/LabelLink.tsx

```diff
@@ -1,24 +1,25 @@
 import React from 'react'
+import { Link } from 'react-router-dom'
 import styled from 'styled-components'
 
 import { Colors } from '../../constants'
 
 interface LabelLinkProps {
-  href?: string
+  to: string
   target?: string
-  children?: string | React.ReactElement
+  children?: React.ReactNode
   className?: string
 }
 
-export function LabelLink({ href, target, children, className }: LabelLinkProps) {
```

Comment:
These changes won't affect anything else because this component is used in only one place - BuyMembershipFormModal